packages/systemd-257/systemd-257.spec

```
@@ -0,0 +1,799 @@
+
+install -d %{buildroot}%{_cross_unitdir}/tmp.mount.d
+install -p -m 0644 %{S:2} %{buildroot}%{_cross_unitdir}/tmp.mount.d/10-no-exec.conf
+
+# Ensure /proc/sys/fs/binfmt_misc mount is wanted by sysinit.target,
+# since we exclude the automount unit.
+ln -s  ../proc-sys-fs-binfmt_misc.mount \
+  %{buildroot}%{_cross_unitdir}/sysinit.target.wants/proc-sys-fs-binfmt_misc.mount
+
+# Remove any README files.
+find %{buildroot} -type f -name README -print -delete
+
+%files
+%license LICENSE.GPL2 LICENSE.LGPL2.1
+%{_cross_attribution_file}
+%{_cross_bindir}/busctl
+%{_cross_bindir}/journalctl
+%{_cross_bindir}/systemctl
+%{_cross_bindir}/systemd-analyze
+%{_cross_bindir}/systemd-ac-power
+%{_cross_bindir}/systemd-cat
+%{_cross_bindir}/systemd-cgls
+%{_cross_bindir}/systemd-cgtop
+%{_cross_bindir}/systemd-creds
+%{_cross_bindir}/systemd-dissect
+%{_cross_bindir}/systemd-delta
+%{_cross_bindir}/systemd-detect-virt
+%{_cross_bindir}/systemd-escape
+%{_cross_bindir}/systemd-id128
+%{_cross_bindir}/systemd-inhibit
+%{_cross_bindir}/systemd-machine-id-setup
+%{_cross_bindir}/systemd-mount
+%{_cross_bindir}/systemd-notify
+%{_cross_bindir}/systemd-nspawn
+%{_cross_bindir}/systemd-path
+%{_cross_bindir}/systemd-repart
+%{_cross_bindir}/systemd-run
+%{_cross_bindir}/systemd-socket-activate
+%{_cross_bindir}/systemd-stdio-bridge
+%{_cross_bindir}/systemd-sysusers
+%{_cross_bindir}/systemd-tmpfiles
+%{_cross_bindir}/systemd-umount
+%{_cross_bindir}/udevadm
+%{_cross_bindir}/loginctl
+%{_cross_bindir}/varlinkctl
+
+%exclude %{_cross_bindir}/run0
+%exclude %{_cross_libdir}/pam.d/systemd-run0
+
+%{_cross_sbindir}/halt
+%{_cross_sbindir}/init
+%{_cross_sbindir}/poweroff
+%{_cross_sbindir}/reboot
+%{_cross_sbindir}/shutdown
+
+%exclude %{_cross_bindir}/systemd-vpick
+%exclude %{_cross_sbindir}/mount.ddi
+
+%{_cross_libdir}/libsystemd.so.*
+%{_cross_libdir}/libudev.so.*
+
+%dir %{_cross_libdir}/systemd
+%{_cross_libdir}/systemd/libsystemd-core-*.so
+%{_cross_libdir}/systemd/libsystemd-shared-*.so
+%{_cross_libdir}/systemd/systemd
+%{_cross_libdir}/systemd/systemd-boot-check-no-failures
+%{_cross_libdir}/systemd/systemd-cgroups-agent
+%{_cross_libdir}/systemd/systemd-executor
+%{_cross_libdir}/systemd/systemd-fsck
+%{_cross_libdir}/systemd/systemd-growfs
+%{_cross_libdir}/systemd/systemd-journald
+%{_cross_libdir}/systemd/systemd-logind
+%{_cross_libdir}/systemd/systemd-makefs
+%{_cross_libdir}/systemd/systemd-modules-load
+%{_cross_libdir}/systemd/systemd-network-generator
+%{_cross_libdir}/systemd/systemd-pstore
+%{_cross_libdir}/systemd/systemd-random-seed
+%{_cross_libdir}/systemd/systemd-remount-fs
+%{_cross_libdir}/systemd/systemd-shutdown
+%{_cross_libdir}/systemd/systemd-sleep
+%{_cross_libdir}/systemd/systemd-socket-proxyd
+%{_cross_libdir}/systemd/systemd-sysctl
+%{_cross_libdir}/systemd/systemd-sysroot-fstab-check
+%{_cross_libdir}/systemd/systemd-udevd
+%{_cross_libdir}/systemd/profile.d/70-systemd-shell-extra.sh
```

exclude since it's a shell script:
```suggestion
%exclude %{_cross_libdir}/systemd/profile.d/70-systemd-shell-extra.sh
```